### ISSUE TITLE
Possibility to manually connect an anime in the anilist list to an anime sought on the servers

### ISSUE THREAD

**Issue by dynamer** [SuperMarcus/NineAnimator#80]
Practically as written in the title, some anime have a different name between the collection server (for example anilist) and the vision server (for example animeunity) in the specific case of these two, selecting Danmachi on anilist where it takes the name of "Dungeon  ni Deai wo Motomeru no wa Machigatteiru Darou ka "and clicking on view episodies can not find the anime in question which on the site is reported as Danmachi.

**Comment by uttiya10** [comment]
This would require having a server that has a database which would stores matches between the two sources, otherwise this feature would only be helpful for yourselves. I don't know if Marcus wants to setup a server, considering the cost to host one.

**Comment by dynamer** [comment]
It would be enough to implement a manual search function for the user (in case no result is found) and then save everything in a json file or in a local database on the device this does not require an online server.

**Comment by uttiya10** [comment]
Yes, that is definitely possible. But, as i said above, that means that only you will access to that json file. Any matching you do, will only be visible to you. Again, its up to marcus to decide if he is okay with that. I just believe it would be better to have the matches available to everyone, so it will improve everyone’s experience.

**Comment by dynamer** [comment]
It would be useful if it were a public database where to insert all the data, but as you said it would be a cost for the developer to bear since the app does not have an internal structure anyway, so at least a local database would already be a step forward in this sense.
Some time ago I used an Android app, very similar to NineAnimator, which allowed this kind of solution, if you want to have a look at it, it's called AnimeDLR.

**Comment by uttiya10** [comment]
Ok lets see what Marcus thinks about this. I personally would like this feature as well.

**Comment by SuperMarcus** [comment]
It's possible to implement something that lets you match `AnimeLink` and `ListingAnimeReference`. We can potentially use something like MALSync to improve matching accuracies. I'll add this to my todo list.

**Comment by uttiya10** [comment]
Can you also add the ability to offset episode numbers:
Some websites like animepahe.com or even Crunchyroll don’t start counting episodes from 1. It depends on what season of anime you are watching. However, MAL always starts counting at 1 no matter what season. This obviously messes stuff up.
